index.yml

```diff
@@ -197,6 +197,12 @@ wsh57:
 webview2:
   Description: Microsoft Edge Web View 2
   Category: Essentials
+powershell:
+  Description: PowerShell Wrapper For Wine
```

Comment:
Minor nit: You may want to specify that it is a Windows PowerShell wrapper. e.g. "Windows PowerShell Wrapper for Wine"